data/json/mapgen/antique_store.json

```diff
@@ -76,13 +76,48 @@
             "t": "f_table"
           },
           "place_items": [
-            { "item": "antique", "x": [ 4, 9 ], "y": [ 10, 10 ], "chance": 70 },
-            { "item": "antique", "x": [ 14, 19 ], "y": [ 10, 10 ], "chance": 70 },
-            { "item": "antique", "x": [ 4, 9 ], "y": [ 13, 13 ], "chance": 70 },
-            { "item": "antique", "x": [ 14, 19 ], "y": [ 13, 13 ], "chance": 70 },
-            { "item": "antique", "x": [ 1, 1 ], "y": [ 8, 15 ], "chance": 70 },
-            { "item": "antique", "x": [ 22, 22 ], "y": [ 8, 15 ], "chance": 70 },
-            { "item": "antique_rare", "x": [ 9, 14 ], "y": [ 16, 16 ], "chance": 65 }
+            {
+              "item": "antique",
+              "x": [ 4, 9 ],
+              "y": [ 10, 10 ],
+              "chance": 70
+            },
```

Comment:
This looks horrible. Mapgens are some of the most object-heavy objects and changing every item entry to at least 5 lines is going to hurt readability a lot.

Reply:
This isn't remotely readable either way. If you can come up with a way to improve it feel free, but trying to forcibly inline objects to conserve lines (what I have to assume you want to happen since you didn't offer a suggestion) isn't a significant improvement.

Reply:
Old version is clearly readable:
* One entry - one line
* "What to place" is in the same place
* No padding with needless newlines
* Everything can be seen at once. Even a big list will not require scrolling.
* Will not bloat diff size to incredible numbers